### ISSUE TITLE
"uniq" is not exported by the List::Util module

### ISSUE THREAD

**Issue by eserte** [shawnlaffan/perl-alien-proj#10]
The test suite seems to fail if List::Util is too old:
```
PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
Bailout called.  Further testing stopped:  Failed to load Alien::proj

#   Failed test 'use Alien::proj;'
#   at t/00-load.t line 10.
#     Tried to use 'Alien::proj'.
#     Error:  "uniq" is not exported by the List::Util module
# Can't continue after import errors at /home/cpansand/.cpan/build/2022032907/Alien-proj-1.23-BqvEsK/blib/lib/Alien/proj.pm line 9.
# BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2022032907/Alien-proj-1.23-BqvEsK/blib/lib/Alien/proj.pm line 9.
# Compilation failed in require at t/00-load.t line 10.
# BEGIN failed--compilation aborted at t/00-load.t line 10.
FAILED--Further testing stopped: Failed to load Alien::proj
Makefile:984: recipe for target 'test_dynamic' failed
make: *** [test_dynamic] Error 255
```

**Comment by shawnlaffan** [comment]
Thanks for the report. 

This should be fixed by ad61bbd9f98b08dbb28d79edf8f9cd649f645938 

Version 1.24 contains it and has just been uploaded to cpan.

**Comment by shawnlaffan** [comment]
CPAN testers is green so closing issue.
